lightly/cli/_helpers.py

```diff
@@ -27,6 +27,15 @@ def print_as_warning(message: str):
     warnings.formatwarning = old_format
 
 
+def available_cpu_count():
+    """Returns the number of CPUs which can be used by the process.
+
+    This number is not equivalent to the number of CPUs in the system.
+
+    """
+    return len(os.sched_getaffinity(0))
```

Comment:
I am unsure, whether this is the best function to use.
According to https://stackoverflow.com/questions/1006289/how-to-find-out-the-number-of-cpus-using-python,
it is only available on linux systems, but we should also target Mac and Windows.

Furthermore, this number might be higher than the actual number of CPU cores, which happens e.g. in the GitHub action:
> /opt/hostedtoolcache/Python/3.7.10/x64/lib/python3.7/site-packages/torch/utils/data/dataloader.py:477: UserWarning: This DataLoader will create 6 worker processes in total. Our suggested max number of worker in current system is 2, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.

 https://github.com/lightly-ai/lightly/runs/2761447843?check_suite_focus=true,